src/framework/vst/vsttypes.h

```diff
@@ -46,6 +46,7 @@
 namespace muse::vst {
 class IVstInstance;
 using IVstInstancePtr = std::shared_ptr<IVstInstance>;
+using VstInstanceId = int;
```

Comment:
What do you think about using muse::ID instead?

Reply:
I don't really like the `muse::ID` class, for example, I don't understand why it has the operators `+`, `+=` for an ID class, this is strange, how can you add two IDs?

and there are also just some inconveniences, `muse::ID`  is more difficult to transfer via interactive, to use in Qml, to set as a model property... (I will need all this)